bylaws.txt

```diff
@@ -90,12 +90,13 @@
 5.2.3.6. Head(s) of Preshows 
 5.2.4. Elections will be held at membership meetings (section 6). Any qualified candidate may nominate themselves for an elected position (5.3). The candidate with the most votes for a given position will be elected.
 5.2.4.1. In the event of a tie, a runoff vote will be held with the tied candidates.
-5.2.4.2. Elections for Director and Financial Secretary shall be held in the February membership meeting. Elections for Heads of standing departments shall be held in the November membership meeting.
-5.2.4.3. Before each election, the Directors shall appoint an active cast member to serve as Election Manager.
-5.2.4.3.1. The Election Manager shall be responsible for conducting the election, ensuring a fair and accurate vote is collected, and reporting the results to the cast at large.
-5.2.4.3.2. The Election Manager may not run in an election they are managing.
-5.2.4.3.3. Any cast member may decline an appointment as Election Manager.
-5.2.4.3.4. The position of Election Manager shall dissolve once the election is concluded and the results reported.
+5.2.4.2. In elections where only one candidate exists for a position, both a "yes" and "no" option will be available. 
```

Comment:
Personally, if somebody puts themself up to run, I think it's kinda a horrible thing that we could say "nah, we'll make the directors pick somebody".  This is a classic small democracy implementation issue, we're small enough that we don't always even have a candidate for leadership positions.  I get the desire for a choice here, we don't want to feel like we're defaulting people into leadership positions, but I do believe that's what we have to do in these situations.  We have to tell people they will be given the chance to lead if they step up for a position and no other cast member will.  If a candidate runs for a position and a majority vote them down the directors are now required to appoint an interim head of the department, from a pool of people who just voted against the only person who volunteered.  The no-confidence process exists in the bylaws already, 5.6.3, and it should be used when an officer has failed to fulfill their duties, but we have to give them that chance.

As written this still conflicts with 5.2.4, it may be best to remove the final sentence entirely from that section and create a new item defining the requirements of election.

In the interest of a second option, I make the following alternative proposal:
>For all elections an option of "No Opinion" shall be given to signal that abstention is a valid choice.

Members would still receive a ballot with two options for unopposed positions.  I think people will feel less pressured to make a choice on elections they don't have a strong feeling on.